source/client/HandView.py

```diff
@@ -37,12 +37,12 @@ def __init__(self, controller, display):
         self.num_wilds = 0
         self.wild_cards = []
         self.selected_list = []
-        self.round_index = 0
+        self.next_round = 0
         self.round_advance = False
         self.ready_color_idx = 2
         self.not_ready_color_idx = 6
         # --- Hand And Foot Specific:
-        self.betweenrounds = ['Welcome to a new game.  This is the round of ' + str(Meld_Threshold[0]) + '.',
+        self.betweenrounds = ['Welcome to a new game.  This is the round of ' + str(Meld_Threshold[self.next_round]) + '.',
```

Comment:
This should have an underscore, and I might call it 'help_text' instead so we know its instructions strings and not some sort of tracker

Reply:
Made suggested change.